.github/workflows/dashboard.yml

```diff
@@ -8,6 +8,8 @@ jobs:
     # This should only run in one place.
     if: contains(github.repository, 'aws/s2n-tls')
     runs-on: ubuntu-latest
+    permissions:
+      contents: write
```

Comment:
Does this work with pull-requests? What does contents refer to?

```suggestion
    permissions:
      pull-requests: write
```

Reply:
Tested and updated to use `pages`; the contents permission was from the action example code...